gap/Utils.gi

```diff
@@ -146,7 +146,7 @@ function(type, alpha, q)
         for delta in GF(q) do
             polynomial := x ^ 2 + delta * x + alpha;
             if IsIrreducibleRingElement(R, polynomial) then
-                result := CoefficientsOfUnivariatePolynomial(Factors(S, polynomial)[1])[1];
+                result := -CoefficientsOfUnivariatePolynomial(Factors(S, polynomial)[1])[1];
```

Comment:
... and that's precisely the kind of issue which makes me suggest that small functions like `CoefficientsOfUnivariatePolynomial` should have their own unit tests :-)